Capstone Project/ViewControllers/Workouts/CurrentWorkoutViewController.m

```diff
@@ -20,6 +25,18 @@ - (void)viewDidLoad {
         self.name.text = self.selectedWorkout.name;
         self.date.date = self.selectedWorkout.date;
     }
+    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    
+    self.arrayOfExercises = [[NSMutableArray alloc] init];
```

Comment:
Why is this object allocation needed if this is going to be just used as a pointer. In line 75, this is being used as a pointer